samples/msal-node-samples/standalone-samples/auth-code/readme.md

```diff
@@ -0,0 +1,203 @@
+# MSAL Node Standalone Sample:  Auth Code
+
+The sample applications contained in this directory are independent samples of MSAL Node usage, covering each of the authorization flows that MSAL Node currently supports. To get started with this sample, first follow the general instructions [here](../readme.me).
+
+Once MSAL Node is installed, and you have the right files, come here to learn about this scenario.
+
+### How is this scenario used?
+The Auth Code flow is most commonly used for a web app that signs in users.  General information about this scenario is available [here](https://docs.microsoft.com/en-us/azure/active-directory/develop/scenario-web-app-sign-user-overview?tabs=aspnetcore).
```

Comment:
Should we mention PKCE here?

Reply:
I meant, this seems to be `confidential client auth-code flow` from the `client-secret` desc below. Are we distinguishing between pca and cca apps here in this section?

Reply:
I didn't include language to distinguish cca and pca here.  Better covered on the docs site, the question is whether a developer would be wondering about that in the middle of this flow.  

I could mention PKCE, but didn't as I see there is another sample with a PKCE label?  We could have a separate PR for that.